com.makemkv.MakeMKV.json

```diff
@@ -0,0 +1,142 @@
+    },
+    {
+      "name": "x264",
+      "config-opts": [
+        "--disable-cli",
+        "--enable-shared",
+        "--disable-asm"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://download.videolan.org/x264/snapshots/x264-snapshot-20181110-2245-stable.tar.bz2",
+          "sha256": "44d843291719dfd340b854c51360fbed1ddcb40580b7a239eccae4ca558c3868"
+        }
+      ]
+    },
+    {
+      "name": "ffmpeg",
+      "config-opts": [
+        "--enable-gpl",
+        "--disable-static",
+        "--enable-shared",
+        "--disable-doc",
+        "--disable-ffplay",
+        "--disable-ffprobe",
+        "--enable-libopus",
+        "--enable-libvpx",
+        "--enable-libx264"
+      ],
+      "post-install": [
+        "chrpath -d /app/bin/ffmpeg"
+      ],
+      "cleanup": [
+        "/share/ffmpeg/examples"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://ffmpeg.org/releases/ffmpeg-4.1.tar.xz",
+          "sha256": "a38ec4d026efb58506a99ad5cd23d5a9793b4bf415f2c4c2e9c1bb444acd1994"
+        }
+      ]
+    },
+    {
+      "name": "makemkv-oss",
+      "buildsystem": "autotools",
+      "cflags": "-D__GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -D __STDC_FORMAT_MACROS",
+      "config-opts": [
+        "--enable-allcodecs"
+      ],
+      "make-install-args": [
+        /* really just to avoid call to ldconfig */
+        "DESTDIR=/"
+      ],
+      "post-install": [
+        "install -Dp -m 644 com.makemkv.MakeMKV.appdata.xml --target-directory=/app/share/appdata"
+      ],
+      "sources" : [
+        {
+          "type": "archive",
+          "url": "http://www.makemkv.com/download/makemkv-oss-1.14.1.tar.gz",
+          "sha256": "fc8e15ef1148988b241110fc7cf2c2d916abf01bb85c3450b4e2ee10aa054b7b"
+        },
+        {
+          "type": "file",
+          "path": "com.makemkv.MakeMKV.appdata.xml"
+        }
+      ]
+    },
+    {
+      "name": "makemkv-bin",
+      "buildsystem": "simple",
+      "build-commands": [
+        /* TODO: put eula in appstream data once it supports it */
+        /* NOTE: author allows redistribution as long as eula is referenced in package description */
+        "mkdir -p tmp",
+        "echo accepted > tmp/eula_accepted",
+        "make install DESTDIR=/ PREFIX=/app"
+      ],
+      "sources" : [
+        {
+          "type": "archive",
+          "url": "http://www.makemkv.com/download/makemkv-bin-1.14.1.tar.gz",
+          "sha256": "17e62cf59c9a8e1e4535d6a6a86cc1b9f6f6c71e86dff8a5ddf38551c2928fd8"
+        }
+      ]
+    },
+    {
+      "name": "ccextractor",
+      /* also supports cmake, but has an odd layout */
+      "buildsystem": "autotools",
+      "subdir": "linux",
+      /* prefer git over a tar archive because the build looks for a git commit */
+      "sources" : [
+        {
+          "type": "git",
+          "url": "https://github.com/CCExtractor/ccextractor",
```

Comment:
Missing `"commit"` key here. Git tags "can change", but as long as you lock it with the commit hash it's 99.9% OK.